packages/react-native-aztec/android/settings.gradle

```diff
@@ -1,2 +1,8 @@
-rootProject.name = '@wordpress/react-native-aztec'
+pluginManagement {
+  plugins {
+    id 'com.jfrog.bintray' version '1.8.5'
```

Comment:
Do you know if there's an automated way to make sure `-aztec` and `-bridge` use the same version of the plugin? That is, something that doesn't require PR reviewers to notice that the version changed in one file but not in the other?

I'm asking this from the assumption that it's safer to have the two projects running on the same version of the plugin.

I think the question will still apply when we'll move away from Bintray, as long as we'll use a plugin to manage the upload.

The only idea I have is a check in CI and/or at the pre-commit level.

Reply:
Nothing I know of, but I do want to look into general dependency version management across Gradle projects. I've seen some ideas about the topic, from some talks by Google or Gradle, but I haven't had the time to check them yet. I also don't know if it'd actually cover this one.

In any case, I think this is outside of the scope of this PR.